Configure.cmake

```diff
@@ -9,6 +9,23 @@ find_library(LIBGMP gmp)
 find_library(LIBRT rt)
 find_library(LIBFFTW3 fftw3)
 
+if (NOT CMAKE_CROSSCOMPILING AND NOT SLEEF_FORCE_FIND_PACKAGE_SSL)
+  find_package(OpenSSL)
+else()
+  # find_package cannot find OpenSSL when cross-compiling
+  find_library(LIBSSL ssl)
+  find_library(LIBCRYPTO crypto)
+  if (LIBSSL AND LIBCRYPTO)
```

Comment:
Why do you need to set OpenSSL libraries as libssl libraries?

Reply:
libssl and libcrypto are the components of OpenSSL library.